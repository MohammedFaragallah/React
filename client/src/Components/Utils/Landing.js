import React from 'react';

const Landing = () => (
  <div>
    <p>welcome here with US</p>
  </div>
);

export default Landing;
