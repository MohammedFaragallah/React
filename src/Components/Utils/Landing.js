import React from 'react';

const Landing = () => {
  return (
    <div>
      <p>welcome here with US</p>
    </div>
  );
};

export default Landing;
