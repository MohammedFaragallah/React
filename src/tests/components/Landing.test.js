import React from 'react';
import { shallow } from 'enzyme';
import Landing from '../../Components/Utils/Landing';

test('should render Landing', () => {
  const wrapper = shallow(<Landing />);
  expect(wrapper).toMatchSnapshot();
});
