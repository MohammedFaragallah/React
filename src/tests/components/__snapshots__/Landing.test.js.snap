// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Landing 1`] = `
<div>
  <p>
    welcome here with US
  </p>
</div>
`;
